src/main/resources/grel_java_mapping.ttl

```diff
@@ -396,6 +410,20 @@ grelm:substringMapping
 
 #LASTINDEXOF
 
+grelm:indexOfMapping
```

Comment:
can you move it between #CONTAINS and #TOLOWERCASE, to be in sync with https://github.com/FnOio/grel-functions-fno/blob/master/grel.ttl , and change the title to #INDEXOF ?